src/lib/components/Feedback.svelte

```diff
@@ -23,7 +28,10 @@
                 email,
                 type: feedbackType,
                 route: $page.route.id,
-                comment
+                comment,
+                metaFields: {
+                    cloudUserId
```

Comment:
It's probably enough to just make this `userId`

Reply:
Done.